_config.yml

```diff
@@ -63,6 +64,14 @@ exclude:
   - vendor/cache/
   - vendor/gems/
   - vendor/ruby/
+  - ANTORA.md
+  - CONTRIBUTING.md
+  - LICENSE.txt
+  - playbook.yml
+  - playbook_author.yml
+  - README.md
+  - yarn.lock
```

Comment:
Do we actually need this file? Are we using Yarn at all even?

Reply:
yeah, shoulda removed - I amended the commit